emission/tests/analysisTests/TestClusterPipeline.py

```diff
@@ -1,171 +0,0 @@
-import unittest
-import emission.core.get_database as edb
-import emission.analysis.modelling.tour_model.cluster_pipeline as cp
-import emission.simulation.trip_gen as tg
-import uuid as uu
-
-class ClusterPipelineTests(unittest.TestCase):
```

Comment:
Note that I have made substantial changes to the codebase in terms of
storing trips and places. Please remember to pull and switch to a new
branch before making any more changes.
https://github.com/e-mission/e-mission-server/pull/265

Shankari